docs/how-linguist-works.md

```diff
@@ -23,7 +23,6 @@ The percentages are calculated based on the bytes of code for each language as r
 
 ## How Linguist works on GitHub.com
 
-When you push changes to a repository on GitHub.com, a low priority background job is enqueued to analyze your repository as explained above.
+When you push changes to a repository on GitHub.com, a low priority background job is enqueued to analyze the default branch of your repository as explained above.
```

Comment:
Semantic line-break:

```suggestion
When you push changes to a repository on GitHub.com,
a low priority background job is enqueued to analyze the default branch of your repository as explained above.
```

It also wouldn't hurt to make _"default branch"_ a link to [GitHub's introduction to default branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches#about-the-default-branch), just for the benefit of readers who're unfamiliar with that term.